_posts/2019-01-29-OAS-3.md

```diff
@@ -0,0 +1,91 @@
+you'd often get hard to understand warnings or errors due to the OpenAPI 2
+parser relying on the JSON Schema from OpenAPI Initiative as the base of
+validations. JSON Schema validations often produce unclear error messages.
+
+We've taken validation completely into our own hands so we can have far greater
+control over the user experience.
+
+As an example, if you miss a required `in` property for a Parameter Object in
+OpenAPI 2, you'll often see a confusing error such as the following:
+
+> Data does not match any schemas from 'oneOf'
+
+You'll see the exact same error message if the value of `in` is not a permitted
+value or is not a string.
+
+Whereas for similar semantic issues, our OpenAPI 3 parser will emit clearer
+errors:
+
+> 'Parameter Object' is missing required property 'in'
+
+> 'Parameter Object' 'in' must be either 'query, 'header', 'path' or 'cookie'"
+
+The state of the errors and warnings in our OpenAPI 2 parser can also fall down
+further when you are dealing with referencing as the entire OAS 2 document is
+dereferenced (stripped of `$ref`) up front, which means that the context of an
+error can be missing and it can be hard to pin-point the exact line which
+caused a parsing error.
+
+### Validation Speed
+
+Another key user experience improvement will be with the performance of our new
+parser in comparison to our OAS 2 parser. We can parse extremely large OAS 3
+documents (multiple megabytes) in an order of magnitude faster than we could
+for OAS 2 so you can get much quicker feedback in our editor.
+
+### Try OpenAPI 3.0 Today
+
+You can try OpenAPI 3.0 today in both Apiary and our open source tools such as
+[Dredd](https://github.com/apiaryio/api-elements.js/issues/71).
+
+When our OpenAPI 3 support reaches maturity, we will be providing you the
+ability to create a new API Project in Apiary using OpenAPI 3. In this stage of
+rollout you can copy in an OpenAPI 3 document into an existing Apiary API
+Project in the Apiary Editor and our editor will detect and support OAS 3
+documents.
+
+We're building our parser out in the open and we welcome any contributions or
+feedback however small, [whether its improving the wording of our warnings and
+error
+messages](https://github.com/apiaryio/api-elements.js/blob/d8d8e292c5240918506fe75d075fbbe406b8d472/packages/fury-adapter-oas3-parser/lib/parser/annotations.js)
+or providing valuable feedback in our [issue
+tracker](https://github.com/apiaryio/api-elements.js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aopenapi3).
+
+You can track the [current status of supported OAS 3
+functionality](https://github.com/apiaryio/api-elements.js/blob/master/packages/fury-adapter-oas3-parser/STATUS.md),
+we are also publishing a [list of the changes we make to our
+parser](https://github.com/apiaryio/api-elements.js/blob/master/packages/fury-adapter-oas3-parser/CHANGELOG.md)
+so you can keep up to date on further developments.
```

Comment:
I'm missing at least a one sentence wrap-up which throws some love on the reader and motivates them to go and try it out.